internal/consensus/state.go

```diff
@@ -1568,20 +1568,20 @@ func (cs *State) enterPrevoteWait(height int64, round int32) {
 
 	if cs.Height != height || round < cs.Round || (cs.Round == round && cstypes.RoundStepPrevoteWait <= cs.Step) {
 		logger.Debug(
-			"entering prevote wait step with invalid args",
+			"Entering prevote wait step with invalid args",
 			"current", log.NewLazySprintf("%v/%v/%v", cs.Height, cs.Round, cs.Step),
 		)
 		return
 	}
 
 	if !cs.Votes.Prevotes(round).HasTwoThirdsAny() {
 		panic(fmt.Sprintf(
-			"entering prevote wait step (%v/%v), but prevotes does not have any +2/3 votes",
+			"Entering prevote wait step (%v/%v), but prevotes does not have any +2/3 votes",
```

Comment:
```suggestion
			"entering prevote wait step (%v/%v), but prevotes does not have any +2/3 votes",
```

Reply:
errors shouldn't be capitalized, just log messages